features/colrv0.yml

```diff
@@ -0,0 +1,7 @@
+name: COLRv0
+description: COLRv0 is a font format that supports multi-color glyphs.
```

Comment:
Can we say something, either here or in v1, that distinguishes the two formats?

Reply:
Updated